src/pymor/core/config.py

```diff
@@ -43,15 +43,19 @@ def _get_dunegdt_version():
     try:
         version = dune.gdt.__version__
         if parse(version) < parse('2021.1.2') or parse(version) >= parse('2021.2'):
-            warnings.warn(f'dune-gdt bindings have been tested for version 2021.1.2 (installed: {dune.gdt.__version__}).')
+            warnings.warn(f'dune-gdt bindings have been tested for version 2021.1.2 '
```

Comment:
The warning doesn't really match the check, right?

Reply:
done in d4888154ffefdb67016ef30cd72dd18957cfd60c